curriculum/scripts/evaluate.py

```
@@ -0,0 +1,182 @@
+        fails.append(f"너무 많이 골랐다: {len(valid)}개 > {cap}개")
+
+    least = exp.get("minSelected")
+    if least is not None and len(valid) < least:
+        fails.append(f"너무 적게 골랐다: {len(valid)}개 < {least}개")
+
+    return fails, {"usage": usage, "ratio": ratio}
+
+
+def run_case(case, courses, model, api_key, max_attempts):
+    budget = case["weeks"] * case["hoursPerWeek"]
+    result, selected, valid, total, problems, attempts, tokens = roadmap.generate(
```

Ruff (0.16.1)

[warning] Unpacked variable `result` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)